packages/orbit-components/src/Tabs/Tabs.stories.tsx

```diff
@@ -65,94 +68,158 @@ export const Compact = () => {
-          Tab 2
-        </Tab>
-        <Tab type="medium" onClick={() => setSelected(2)} active={selected === 2}>
-          Tab 3
-        </Tab>
-        <Tab type="top" onClick={() => setSelected(3)} active={selected === 3}>
-          Tab 4
-        </Tab>
-      </TabList>
-      <TabPanels>
-        <TabPanel active={selected === 0}>Tab 1 content</TabPanel>
-        <TabPanel active={selected === 1}>Tab 2 content</TabPanel>
-        <TabPanel active={selected === 2}>Tab 3 content</TabPanel>
-        <TabPanel active={selected === 3}>Tab 4 content</TabPanel>
-      </TabPanels>
-    </Tabs>
-  );
+  parameters: {
+    info: "This is the default setup of Tabs component. Check Orbit.kiwi for more detailed guidelines.",
+    controls: {
+      disable: true,
+    },
+  },
 };
 
-Default.story = {
-  name: "Default",
-};
+export const Controlled: Story = {
+  render: () => {
+    const [selected, setSelected] = React.useState(2);
 
-export default {
-  title: "Tabs",
-};
+    return (
+      <Tabs>
+        <TabList>
+          <Tab onClick={() => setSelected(0)} active={selected === 0}>
+            Tab 1
+          </Tab>
+          <Tab type="basic" onClick={() => setSelected(1)} active={selected === 1}>
+            Tab 2
+          </Tab>
+          <Tab type="medium" onClick={() => setSelected(2)} active={selected === 2}>
+            Tab 3
+          </Tab>
+          <Tab type="top" onClick={() => setSelected(3)} active={selected === 3}>
+            Tab 4
+          </Tab>
+        </TabList>
+        <TabPanels>
+          <TabPanel active={selected === 0}>Tab 1 content</TabPanel>
+          <TabPanel active={selected === 1}>Tab 2 content</TabPanel>
+          <TabPanel active={selected === 2}>Tab 3 content</TabPanel>
+          <TabPanel active={selected === 3}>Tab 4 content</TabPanel>
+        </TabPanels>
+      </Tabs>
+    );
+  },
 
-export const Playground = ({
-  type,
-  compact,
-  disabled,
-  defaultSelected,
-  firstTabLabel,
-  secondTabLabel,
-  firstTabContent,
-  secondTabContent,
-  fullWidth,
-}) => {
-  return (
-    <Tabs onChange={action("onChange")} defaultSelected={defaultSelected}>
-      <TabList compact={compact} fullWidth={fullWidth}>
-        <Tab disabled={disabled}>{firstTabLabel}</Tab>
-        <Tab type={type}>{secondTabLabel}</Tab>
-      </TabList>
-      <TabPanels>
-        <TabPanel>{firstTabContent}</TabPanel>
-        <TabPanel>{secondTabContent}</TabPanel>
-      </TabPanels>
-    </Tabs>
-  );
+  parameters: {
+    info: "Example of controlled version of Tabs component. Check Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      disable: true,
+    },
+  },
 };
 
-Playground.args = {
-  type: TYPE_OPTIONS.DEFAULT,
-  compact: false,
-  disabled: false,
-  defaultSelected: 0,
-  firstTabLabel: "Tab 1",
-  secondTabLabel: "Tab 2",
-  firstTabContent: "Tab 1",
-  secondTabContent: "Tab 2",
-  fullWidth: false,
-};
+export const Playground: Story = {
```

Comment:
In this `Playground` story I didn't include `active` and `onClick` props for `Tab`/`TabPanel` component as it's related to the Controlled state. In case when have two tabs in the playground, I couldn't find a simpler solution to handle the `active` state separately for each Tab/TabPanel.